cli/cmd/customer_update.go

```diff
@@ -62,28 +67,49 @@ func (r *runners) updateCustomer(cmd *cobra.Command, _ []string) (err error) {
 		r.args.customerUpdateType = "prod"
 	}
 
-	channelID := ""
-	if r.args.customerUpdateChannel != "" {
+	channels := []kotsclient.CustomerChannel{}
+
+	foundDefaultChannel := false
+	for _, requestedChannel := range r.args.customerUpdateChannel {
 		getOrCreateChannelOptions := client.GetOrCreateChannelOptions{
 			AppID:          r.appID,
 			AppType:        r.appType,
-			NameOrID:       r.args.customerUpdateChannel,
+			NameOrID:       requestedChannel,
 			Description:    "",
-			CreateIfAbsent: r.args.customerUpdateEnsureChannel,
+			CreateIfAbsent: r.args.customerCreateEnsureChannel,
 		}
 
 		channel, err := r.api.GetOrCreateChannelByName(getOrCreateChannelOptions)
 		if err != nil {
 			return errors.Wrap(err, "get channel")
 		}
 
-		channelID = channel.ID
+		customerChannel := kotsclient.CustomerChannel{
+			ID: channel.ID,
+		}
+
+		if r.args.customerCreateDefaultChannel == requestedChannel {
```

Comment:
`customerCreateDefaultChannel` can be an empty string. This should be allowed only when single channel is specified, which should always have `IsDefault` set.